docker/base/base-core

```diff
@@ -102,6 +98,9 @@ RUN mv apache-maven-* /usr/local/apache-maven
 # define environment variables for maven
 ENV M2_HOME /usr/local/apache-maven
 ENV PATH $PATH:/usr/local/apache-maven/bin
+# copy local maven repository to docker image
+#RUN rm -rf /root/.m2
+#ADD .m2 /root/.m2
```

Comment:
originally , I think these three lines will provide some hints for users who don't have good network environment. Because downloading maven repositories is a time-consuming process. Maybe you are right, It's not meaningful enough. I can remove these.